src/Mono.Android/Test/Mono.Android-Tests.projitems

```diff
@@ -16,8 +16,8 @@
   </ItemGroup>
   <Target Name="_RecordApkSizes" AfterTargets="DeployUnitTestApks">
     <Delete Files="$(MSBuildThisFileDirectory)..\..\..\TestResult-Mono.Android_Tests-values.csv" Condition=" '$(Configuration)' == 'Debug' " />
-    <Exec Command="stat -f &quot;stat: %z %N&quot; &quot;$(_MonoAndroidTestApkFile)&quot; > $(_MonoAndroidTestApkSizesInput)" />
-    <Exec Command="unzip -l &quot;$(_MonoAndroidTestApkFile)&quot; >> $(_MonoAndroidTestApkSizesInput)" />
+    <Exec Condition=" '$(HostOS)' != 'Linux' " Command="stat -f &quot;stat: %z %N&quot; &quot;$(_MonoAndroidTestApkFile)&quot; > $(_MonoAndroidTestApkSizesInput)" />
```

Comment:
This should probably be `== 'Darwin'`. A (soon-ish!) goal is to be able to run unit tests on Windows, and `stat` isn't going to work there...